cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,41 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-describe('Article', () => {
-  before(() => {
+import ArticlePageObject from '../support/pages/article.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
 
-  });
+const articlePage = new ArticlePageObject();
+const signInPage = new SignInPageObject();
 
-  beforeEach(() => {
-    cy.task('db:clear');
+describe('Article', () => {
+  before(() => {
+    cy.visit('/');
+    signInPage.visit();
+    signInPage.typeEmail('riot@qa.team');
+    signInPage.typePassword('12345Qwert!');
+    signInPage.clickSignInBtn();
   });
 
   it('should be created using New Article form', () => {
-
+    articlePage.createNewArticle();
```

Comment:
Consider using dynamic data for creating new articles to avoid conflicts if the test is run multiple times. You can use libraries like `faker` to generate random titles and descriptions.